### ISSUE TITLE
prediction results are not been shown in the project directory

### ISSUE THREAD

**Issue by SandipanCh93** [ultralytics/ultralytics#20346]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Dear all,

I am running a pretrained yolo8 model, ran the predictions using the docker container, the console is letting me know that the image is getting saved in the specific path, but I am unable to see in the explorer, anybody can help me with this?

<img width="959" alt="Image" src="https://github.com/user-attachments/assets/66117277-3542-4594-ac4f-77337654b221" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @SandipanCh93, thank you for reaching out and using Ultralytics 🚀! This is an automated response to help get you started—an Ultralytics engineer will assist you soon.

We recommend visiting the [Docs](https://docs.ultralytics.com/) for detailed guides and checking out [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, which often address common issues like saving prediction results.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug your issue—especially helpful when working inside Docker containers.

If this is a custom usage ❓ Question, please include as much detail as possible, such as your exact Docker command, the output paths shown in the console, and any relevant logs or screenshots. Also, check our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure your setup aligns with best practices.

Join the Ultralytics community for faster support and collaboration:
- Real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions: [Discourse](https://community.ultralytics.com/)
- Community threads: [Subreddit](https://reddit.com/r/Ultralytics)

## Upgrade

Please ensure you are running the latest `ultralytics` package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in any of these up-to-date, verified environments (with dependencies such as [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM ([GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon** Deep Learning AMI ([AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker Image** ([Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you for your patience! Please provide any additional details or a minimum reproducible example to help us address your issue.

**Comment by Y-T-G** [comment]
You're passing a path as `project`. It should be a name only.

**Comment by SandipanCh93** [comment]
Thanks for the reply, it worked..:)

**Comment by glenn-jocher** [comment]
I'm glad to hear the issue is resolved! Just for additional clarity for others who might encounter this: when using the `project` parameter, it should be a name (like "runs/detect") rather than a complete file path. The Ultralytics framework automatically handles directory creation and organization within that project folder. If you need to view the saved results from a Docker container, you may also want to ensure you've mounted the appropriate volumes to access those files from your host system.
